docs/scale/02-deploy-infra.md

```diff
@@ -0,0 +1,218 @@
+<!--[metadata]>
++++
+title = "Deploy network infrastructure"
+description = "Try Swarm at scale"
+keywords = ["docker, swarm, scale, voting, application,  certificates"]
+[menu.main]
+parent="scale_swarm"
+weight=-90
++++
+<![end-metadata]-->
+
+# Deploy your infrastructure
+
+In this step, you create an AWS Virtual Private Cloud (VPC) to run your
+application stack on. Before you continue, make sure you have taken the time to
+[learn the application architecture](01-about.md).
+
+This example uses AWS but the AWS provider is only one example of an
+infrastructure you can use. You can create the environment design on whatever
+infrastructure you wish. For example, you could place the application on another
+public cloud platform such as Azure or DigitalOcean, on premises in your data
+center, or even in in a test environment on your laptop.
+
+>**Note**: If you are not deploying to AWS, or are not using the CloudFormation
+template used in the instructions below, make sure your Docker hosts are running
+a 3.16 or higher kernel. This kernel is required by Docker's container
+networking feature.
+
+## Overview of the deployment process
+
+To deploy on an AWS infrastructure, you first build a VPC and then apply apply
+the [CloudFormation template](https://github.com/docker/swarm-microservice-demo-v1/blob/master/AWS/cloudformation.json) prepared for you. The template describes the hosts in the example's stack. While you
+could create the entire VPC and all instances via a CloudFormation template,
+splitting the deployment into two steps lets you use the CloudFormation template
+to build the stack on an *existing VPCs*.
+
+The diagram below shows the VPC infrastructure required to run the
+CloudFormation template.
+
+![](../images/cloud-formation-tmp.jpg)
+
+The configuration is a single VPC with a single public subnet. The VPC must be
+in the `us-west-1` Region (N. California). This region is **required** for this
+particular CloudFormation template to work.
+
+The VPC network address space is `192.168.0.0/16` and single 24-bit public
+subnet is carved out as 192.168.33.0/24. The subnet must be configured with a
+default route to the internet via the VPC's internet gateway. All six EC2
+instances are deployed into this public subnet.
+
+Once the VPC is created, you deploy the EC2 instances using the
+CloudFormation template located
+[in the `docker/swarm-microservice-demo-v1` repository](https://github.com/docker/swarm-microservice-demo-v1/blob/master/AWS/cloudformation.json).
+
+## Prerequisites
+
+You'll need to have an Amazon AWS account. This account can be personal or
+through a corporate instance. The account must be able to deploy EC2 instances
+in the `us-west-1` region (N. California).
```

Comment:
Should we mention if user's region is different, what need to be changed to in the scripts?